pages/linux/blastn.md

```diff
@@ -0,0 +1,28 @@
+# blastn
+
+> Nucleotide-Nucleotide BLAST.
+> More information: <https://www.ncbi.nlm.nih.gov/books/NBK279696/>.
+- Print USAGE and DESCRIPTION;  ignore all other parameters:
+
+`blastn -h`
+
+- Align two or more sequences using megablast (default), with the e-value threshold of 1e-9, pairwise output format (default):
+
+`blastn -query {{query.fa}} -subject {{subject.fa}} -evalue 1e-9`
```

Comment:
Is evalue threshold a user-inputted value if so then it must be a command argument enclosed in {{}}?

Reply:
Yes, it is a user-inputted value. So it should be enclosed in {{}}

Reply:
`blastn -query {{query.fa}} -subject {{subject.fa}} -evalue {{1e-9}}`

Reply:
LGTM, We can commit this then
```suggestion
`blastn -query {{query.fa}} -subject {{subject.fa}} -evalue {{1e-9}}`
```